### PR TITLE
Implement new webapp

### PR DESCRIPTION
**Pull request by tetogomez**
### What does this PR do?

- Implement new webapp

### Steps to test

1. first step
2. next step

#### CheckList

- [] Follow proper Markdown format
- [] The content is adequate
- [] The content is available in both english and spanish
- [] I Ran a spell check

### CODE REVIEW

**Review by AngeloCG97 — APPROVED** [review]
LGTM

File: webapp/src/hooks/useUAL/index.tsx [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+                authenticator,
+              })
+            }
+          }
+        }, 250)
+      }
+    } catch (e: any) {
+      clearCache()
+      const msg = 'sessionEndedNeedLogin'
+      const source = type || 'Universal Authenticator Library'
```
[the remark this answers is not in the document]
Comment (xavier506) [reply webapp/src/hooks/useUAL/index.tsx:210]: @tetogomez what do you think about this comment?

[the remark this answers is not in the document]
Comment (tetogomez) [reply webapp/src/hooks/useUAL/index.tsx:210]: it's necessary for TS validation

image

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**